### PR TITLE
update metadata

### LINKED ISSUE #830: Update MANIFEST.in
**Is your feature request related to a problem? Please describe.**

`MANIFEST.in` should be updated.

**Describe the solution you'd like**

Update according to [this](https://packaging.python.org/guides/using-manifest-in/)

**Describe alternatives you've considered**

**Additional context**

### PR DESCRIPTION
**Pull request by DavidMinarsch**
## Proposed changes

Fixes some metadata, including Manifest.in

## Fixes

#830

## Types of changes

What types of changes does your code introduce to agents-aea?
_Put an `x` in the boxes that apply_

- [x] Bugfix (non-breaking change which fixes an issue)
- [ ] New feature (non-breaking change which adds functionality)
- [ ] Breaking change (fix or feature that would cause existing functionality to not work as expected)

## Checklist

_Put an `x` in the boxes that apply._

- [x] I have read the [CONTRIBUTING](../CONTRIBUTING.md) doc
- [x] I am making a pull request against the `develop` branch (left side). Also you should start your branch off our `develop`.
- [x] Lint and unit tests pass locally with my changes
- [x] I have added tests that prove my fix is effective or that my feature works
- [x] I have checked that code coverage does not decrease.
- [x] I have checked that the documentation about the `aea cli` tool works
- [x] I have added necessary documentation (if appropriate)
- [x] Any dependent changes have been merged and published in downstream modules

## Further comments

-

### CODE REVIEW

File: setup.py [hunk 1]
```diff
@@ -124,16 +124,26 @@ def get_all_extras() -> Dict:
...
         'Programming Language :: Python :: 3.8',
+        'Topic :: Communications',
+        'Topic :: Internet',
+        'Topic :: Scientific/Engineering',
+        'Topic :: Software Development',
+        'Topic :: System',
     ],
     install_requires=base_deps,
-    tests_require=["tox"],
+    tests_require=['tox'],
```
Comment (marcofavorito) [inline setup.py]: was this by purpose? If yes, why not changing the other strings below from "double-quoted" to 'single-quoted'?

File: setup.cfg [hunk 2]
```diff
@@ -15,32 +15,12 @@ select = C,E,F,I,W,B901,B902,B903,B301,B302,B303,B304,B305,B306,B001,B002,B003,B
 ignore = E203,E501,W503,D202,B009,B014
```
Comment (marcofavorito) [inline setup.cfg]: why have some parts been removed?

File: MANIFEST.in [hunk 3]
```diff
@@ -1,3 +1,4 @@
-include README.md LICENSE Pipfile Pipfile.lock
+include README.md LICENSE HISTORY.md AUTHORS.md Pipfile mkdocs.yml tox.ini pytest.ini strategy.ini
 
-recursive-include aea *
+recursive-include docs *
+recursive-include tests *
```
Comment (marcofavorito) [inline MANIFEST.in]: `black` checks from `tox` fail because `examples`, `packages` and `scripts` are not included.

Comment (marcofavorito) [reply MANIFEST.in]: for analogous reasons, `mypy` and `liccheck` fail.

File: setup.py (hunk 1 above)
Comment (marcofavorito) [reply setup.py]: consider also the classifier strings above.

File: setup.cfg (hunk 2 above)
Comment (DavidMinarsch) [reply setup.cfg]: because they were no longer needed

**Pull request merged**